src/SSHDebugPS/StringResources.resx

```diff
@@ -162,6 +162,9 @@
     <value>Source file '{0}' not found.</value>
     <comment>{0} = source file name</comment>
   </data>
+  <data name="Error_DockerTypesNotMatching" xml:space="preserve">
```

Comment:
change the error name to be more indicative of the actual message. maybe "Error_EnsureDockerContainerIsLinux" or something.  Also use `Ensure` versus `Make sure`.